test/functional/rpc_createmultisig.py

```diff
@@ -117,13 +133,13 @@ def do_multisig(self, nkeys, nsigs, output_type, wallet_multi):
             assert mredeemw == mredeem
 
         spk = address_to_scriptpubkey(madd)
-        value = decimal.Decimal("0.00001300")
+        value = decimal.Decimal("0.00004000")
         tx = self.wallet.send_to(from_node=self.nodes[0], scriptPubKey=spk, amount=int(value * COIN))
         prevtxs = [{"txid": tx["txid"], "vout": tx["sent_vout"], "scriptPubKey": spk.hex(), "redeemScript": mredeem, "amount": value}]
 
         self.generate(node0, 1)
 
-        outval = value - decimal.Decimal("0.00001000")
+        outval = value - decimal.Decimal("0.00002000")  # deduce fee (must be higher than the min relay fee)
```

Comment:
In 9be6065cc03f2408f290a332b203eef9c9cebf24 "test: coverage for 16-20 segwit multisig scripts"

nit: s/deduce/deduct